app/components/member/Edit/form.js

```diff
@@ -12,8 +12,8 @@ import OrganizationMemberUpdateMutation from '../../../mutations/OrganizationMem
 
 import OrganizationMemberRoleConstants from '../../../constants/OrganizationMemberRoleConstants';
 
-class Role extends React.PureComponent {
-  static displayName = "Member.Edit.Role";
+class Form extends React.PureComponent {
+  static displayName = "Member.Edit.Form";
```

Comment:
I’m wondering if the `displayName = blah` pattern is the wrong approach here. Given the Relay compiler seems to insist component names are their file name. It’s real irritating :(